UPGRADE.md

```diff
@@ -6,11 +6,14 @@ Since these are 3 possible scenarios how you can use shopsys, instructions are d
 Follow instructions in relevant sections, eg. `shopsys/coding-standards`, `shopsys/microservice-product-search`.
 
 ### You are using monorepo
-Follow instructions in the section `shopsys/shopsys`.
+Follow instructions in a [monorepo upgrade guide](./docs/contributing/UPGRADE.md)
 
 ### You are developing a project based on project-base
 * upgrade only your composer dependencies and follow instructions
-* if you want update your project with the changes from [shopsys/project-base], you can follow the *(optional)* instructions or cherry-pick from the repository whatever is relevant for you but we do not recommend rebasing or merging everything because the changes might not be compatible with your project as it probably evolves in time
+* if you want update your project with the changes from [shopsys/project-base],
```

Comment:
after reading the whole documentation, I didn't know what instructions are relevant for upgrading project-base. I would recommend to move the bullet `check all instructions in all sections, any of them could be relevant for you` to second place (after `upgrade only your composer dep...`), so it has "higher priority" for reader

Reply:
Moved, thanks 👍